inference-engine/thirdparty/clDNN/api/cldnn/graph/network.hpp

```diff
@@ -66,6 +66,9 @@ struct network {
 
     network(program::ptr program, stream::ptr stream, uint16_t stream_id);
 
+    //network(const network&) = delete;
+    //network& operator=(const network&) = delete;
```

Comment:
why? If it's really needed, please remove these lines

Reply:
ah they are to be removed...